app/assets/javascripts/study_level_activities.js.coffee

```diff
@@ -83,12 +85,17 @@ $ ->
       data: "line_item_id" : line_item_id
 
   $(document).on 'click', '.otf_fulfillment_edit', ->
+    allowSubmit = true
     row_index   = $(this).parents("tr").data("index")
     fulfillment_id = $(this).parents("#fulfillments-table").bootstrapTable("getData")[row_index].id
     $.ajax
       type: 'GET'
       url: "/fulfillments/#{fulfillment_id}/edit"
 
-  $(document).on 'click', '.add_fulfillment', ->
-    $('.add_fulfillment').prop('disabled', true)
-    $('.fulfillment-form').submit()
+  $(document).on 'click', '.add_fulfillment', (e)->
+    e.preventDefault()
+    if allowSubmit
```

Comment:
Would the `data-disable-with` option on the button work for this?

https://stackoverflow.com/questions/21812230/rails-4-disable-submit-button-after-click

Reply:
Yeah, I saw that solution and tried it. Did not work.